docs/get_start.rst

```diff
@@ -169,3 +169,30 @@ BOX configuration can be refered on `Packer BOX Configuration <https://github.co
         Network Adapter: Bridged Adapter
         Promiscuous Mode: on
 
+Relationship table of Infrasim command and standard server command
+--------------------------------------------------------------------
+
+Here we list a table to reflect the relationship of InfraSIM command and standard server command.
+
+    +---------------------------------------------+-----------------------------+
+    |standard server command                      |InfraSIM command             |
+    +=============================================+=============================+
+    |power on                                     |infrasim node start          |
+    +---------------------------------------------+-----------------------------+
+    |power off                                    |infrasim node stop           |
+    +---------------------------------------------+-----------------------------+
+    |removal server                               |infrasim node destroy        |
+    +---------------------------------------------+-----------------------------+
+    |reset                                        |infrasim node restart        |
+    +---------------------------------------------+-----------------------------+
+    |KVM - virtual keyboard, visual monitor       |infrasim VNC                 |
+    +---------------------------------------------+-----------------------------+
+    |configuration update                         |1. update yaml               |
```

Comment:
"configuration update for a-node"
1. update /a/b/c/a-node.yml"
2.  ...
3. ...
4. ...